inf-terraform-aws/Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+def odsNamespace = env.ODS_NAMESPACE ?: 'ods'
+def odsGitRef = env.ODS_GIT_REF ?: 'master'
+def odsImageTag = env.ODS_IMAGE_TAG ?: 'latest'
+
+library("ods-jenkins-shared-library@${odsGitRef}")
+
+odsQuickstarterPipeline(
+  imageStreamTag: "${odsNamespace}/jenkins-agent-base:${odsImageTag}",
+) { context ->
+
+  odsQuickstarterStageCopyFiles(context)
+
+  odsQuickstarterStageCreateOpenShiftResources(
```

Comment:
Do you really want this? I'd think no OpenShift resources are required ...

Reply:
copy paste error I guess ... removed it 254fc6d